.github/workflows/merge-master.yml

```diff
@@ -42,25 +42,9 @@ jobs:
 
       - name: Build Docker Image
         run: |
-          make docker repo=${{ secrets.RELEASE_REPO }}
-          make docker tag=latest
-
-      - name: helm version
-        id: helm
-        run: |
-          HELM_VERSION=$(make version)
-          echo ::set-output name=version::"$HELM_VERSION"
-
-      - name: package helm
-        run: |
-          make helm version=${{ steps.helm.outputs.version }}
-
-      - name: Publish Release Chart 
-        id: upload
-        uses: google-github-actions/upload-cloud-storage@main
-        with:
-          path: ${{ env.SERVICE_NAME }}-${{ steps.helm.outputs.version }}.tgz
-          destination: ${{ secrets.HELM_RELEASE_BUCKET }}/${{ env.SERVICE_NAME }}
+          make docker tag=${{ github.sha }}
+          docker tag eu.gcr.io/${{ secrets.DEV_REPO }}/securebanking/${{ env.SERVICE_NAME }}:{{ github.sha }} eu.gcr.io/${{ secrets.DEV_REPO }}/securebanking/${{ env.SERVICE_NAME }}:latest
+          docker push eu.gcr.io/${{ secrets.DEV_REPO }}/securebanking/${{ env.SERVICE_NAME }}:latest
```

Comment:
The tag github.sha should not also be uploaded?

- docker push eu.gcr.io/${{ secrets.DEV_REPO }}/securebanking/${{ env.SERVICE_NAME }}:{{ github.sha }}

Reply:
`make docker` does an implicit push so this step will push both git sha and latest